### ISSUE TITLE
Alias doesn't work with ForeignKey / Raw ID widget / selection pop-ups

### ISSUE THREAD

**Issue by krmax44** [django-cms/djangocms-alias#246]
I have a model that has a foreign key to an alias and a Raw ID widget in my admin panel. When I click on the lens icon to select an alias, I can't select anything in the popup window. Usually, each row would be a link and clicking that would insert the id into the field.

![image](https://github.com/user-attachments/assets/f397bb3b-5267-48fa-8c86-4684f7122ca1)

But there is no link:

![image](https://github.com/user-attachments/assets/c24d0fb1-ff9d-402d-83c8-517380f24fd2)

**Comment by fsbraun** [comment]
@krmax44 Indeed, that is correct. I suggest using an autocomplete field instead.
